A.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hello Microverse</title>
+  <link rel="stylesheet" href="A.css">
+</head>
+
+<body>
+  <h1>Hello Microverse!</h1>
+</body>
```

Comment:
- Nice job using `h1` tags for the text, please add a `header` which is better for the [semantics](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) of your HTML file.